charts/nginx-echo-headers/values.yaml

```diff
@@ -13,6 +13,17 @@ image:
   # Overrides the image tag whose default is the chart appVersion.
   tag: "0.0.2"
 
+ingress:
```

Comment:
I'm wondering if we should consider renaming `ingress` to `traefik`. Only because if other people look at this chart they may think this is the native Kubernetes `Ingress` resource? Ok either way though.

Reply:
good point! I can rename it 👍 Thanks @hawkesn